gradle.properties

```diff
@@ -1,6 +1,8 @@
 # Gradle properties
 org.gradle.jvmargs=-Xmx3G
 org.gradle.daemon=false
+org.gradle.caching=true 
+org.gradle.parallel = true
```

Comment:
These settings break on some devs machines. Specifically parallel likes to fail on macs.